docs/pages/reference/config.mdx

```diff
@@ -640,7 +640,7 @@ proxy_service:
     #
     # If not set, behavior depends on the config file version:
     #
-    # "v2": listener is not created, SSH is multiplexed on web_listen_addr
+    # "v2" & onwards: listener is not created, SSH is multiplexed on web_listen_addr
```

Comment:
I'm not a huge fan of

```
"v2" & onwards:
```

But I just added ` & onwards` to what was there before. I'm open to changing these to remove the double quotes, it might look less weird

Reply:
I think "and" would be slightly easier to understand than an ampersand, though maybe that's just me! Also, I think "above" is more explicit than "onwards" here.